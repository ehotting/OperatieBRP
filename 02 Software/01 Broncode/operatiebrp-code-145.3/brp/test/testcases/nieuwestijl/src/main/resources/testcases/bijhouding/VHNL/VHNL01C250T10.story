Meta:
@status                 Klaar
@sleutelwoorden         voltrekkingHuwelijkInNederland VHNL01C250T10
@usecase                UCS-BY.HG

Narrative:
R2146 Land/gebied geboorte is verplicht bij Niet-ingeschrevene of Onbekend Persoon

Scenario:   Land/gebied geboorte leeg bij Onbekend Persoon
            LT: VHNL01C250T10



Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL/Libby.xls

When voer een bijhouding uit VHNL01C250T10.xml namens partij 'Gemeente BRP 1'

Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding/VHNL/expected/VHNL01C250T10.txt

Then lees persoon met anummer 1868196961 uit database en vergelijk met expected VHNL01C250T10-persoon1.xml









