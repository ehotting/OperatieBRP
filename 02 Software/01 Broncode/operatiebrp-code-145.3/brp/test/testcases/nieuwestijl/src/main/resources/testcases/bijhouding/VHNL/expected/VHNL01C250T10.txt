cvc-complex-type.2.4.b: The content of element 'brp:geboorte' is not complete. One of '{"http://www.bzk.nl/brp/brp0200":buitenlandsePlaats, "http://www.bzk.nl/brp/brp0200":buitenlandseRegio, "http://www.bzk.nl/brp/brp0200":omschrijvingLocatie, "http://www.bzk.nl/brp/brp0200":landGebiedCode}' is expected.